src/gt4py/next/program_processors/runners/dace_iterator/utility.py

```diff
@@ -35,6 +35,20 @@ def as_dace_type(type_: ts.ScalarType):
     raise ValueError(f"scalar type {type_} not supported")
 
 
+def as_scalar_type(dtype: str) -> ts.ScalarType:
+    if dtype == "bool":
+        return ts.ScalarType(kind=ts.ScalarKind.BOOL)
+    elif dtype == "float32":
+        return ts.ScalarType(kind=ts.ScalarKind.FLOAT32)
+    elif dtype == "float64":
+        return ts.ScalarType(kind=ts.ScalarKind.FLOAT64)
+    elif dtype == "int32":
+        return ts.ScalarType(kind=ts.ScalarKind.INT32)
+    elif dtype == "int64":
+        return ts.ScalarType(kind=ts.ScalarKind.INT64)
+    raise ValueError(f"data type {dtype} not supported")
```

Comment:
```suggestion
    try:
      return getattr(ts.ScalarKind, dtype.upper())
    except AttributeError:
          raise ValueError(f"Data type {dtype} not supported.")
```

Reply:
Thank you, nice support for reflection in Python language!